.github/actions/e2e/create-cluster/alpha-controller-policy.json

```diff
@@ -163,4 +163,4 @@
       }
```

Comment:
Can we drop the alpha controller policy and the instance profile creation if we are no longer testing the alpha behavior?